### PR TITLE
Add PR's to CHANGES

### PR DESCRIPTION
**Pull request by airween**
## what

Add PR's to `CHANGES`

## why

PR's weren't added during the merge process.

### CODE REVIEW

**Comment by airween** [comment]
Do not approve nor merge this yet.

**Comment by airween** [comment]
I think it's done, I do not want to add anything else. I removed [do not merge] label.

File: CHANGES [hunk 1]
```diff
@@ -1,6 +1,93 @@
...
+   [Issue #3159 - @fzipi]
+ * chore: add gitignore file
+   [Issue #3158 - @fzipi]
+ * Possible double free
+   [Issue #3155 - @marcstern]
+ * Set 'jit' variable's initial value
+   [Issue #3154 - @marcstern]
+ * Missing null byte + optimization
+   [Issue #3153 - @marcstern]
+ * fix: remove usage of insecure tmpnam
```
Comment (theseion) [inline CHANGES]: `tmpname`?

File: CHANGES [hunk 2]
```diff
@@ -1,6 +1,93 @@
...
+   [Issue #3149 - @fzipi]
+ * docs: update copyright
+   [Issue #3148 - @fzipi]
+ * Enhanced logging [Issue #3107]
+   [Issue #3139 - @marcstern]
+ * Check for null pointer dereference (almost) everywhere
+   [Issue #3120 - @marcstern]
+ * Fix possible segfault in collection_unpack
+   [Issue #3099 - @twouters]
+ * fix: Replace obsolote macros
```
Comment (theseion) [inline CHANGES]:
```suggestion
 * fix: Replace obsolete macros
```

**Review by theseion — CHANGES_REQUESTED** [review]

**Comment by theseion** [comment]
Maybe for the next release, consider going the same way as CRS and dropping the changes file in favour of the auto generated changelog through GitHub releases.

File: CHANGES (hunk 1 above)
Comment (airween) [reply CHANGES]: Thanks - I generated CHANGES through GH API, and picked up descriptions from PR's. Or it can be a single typo by me :smile:

**Comment by airween** [comment]
> Maybe for the next release, consider going the same way as CRS and dropping the changes file in favour of the auto generated changelog through GitHub releases.

Thanks, we need to inspect this possibility. I added your suggestions - thanks.

**Review by theseion — APPROVED** [review]

**Pull request merged**